### ISSUE TITLE
Stuck at Apple logo

### ISSUE THREAD

**Issue by icydiafire** [SchmockLord/Hackintosh-Intel-i9-10900k-Gigabyte-Z490-Vision-D#169]
I copied your EFI and just changed the SMbios with mine iMacPro1,1 

Set the XHCIportlimit to false otherwise I can not use my Bluetooth keyboard & Mouse

Checked it with the Plist checker seems everything is ok. So could you please help?? 

![image](https://user-images.githubusercontent.com/7584627/121347159-2533ee80-c927-11eb-9c4b-53ed4cc68790.jpeg)

**Comment by SchmockLord** [comment]
Yeah, that's the issue with using USB-Map. It generates a kext that is bound to your SMBIOS. That's why I have one for iMacPro1,1 and iMac20,2.

**Comment by icydiafire** [comment]
But i can't boot from both of them, what can I do?

**Comment by icydiafire** [comment]
i am using the iMacPro1,1 and i meant that i changed to my own serial number, udid etc, i didnt changed the iMac20,2 or something like that... so i just opened the plist of the iMacPro1,1 and put my own generated Serial and Udid, MB...

**Comment by icydiafire** [comment]
[EFI.zip](https://github.com/SchmockLord/Hackintosh-Intel-i9-10900k-Gigabyte-Z490-Vision-D/files/6626693/EFI.zip)
Could you please check it asap..
